### DISCUSSION TITLE
Get files from chiseled image running in K8s as container

### DISCUSSION THREAD

**Discussion by McMlok** [dotnet/dotnet-docker#5210] [Q&A] [answered]
Hello,

I'm using .NET 8 jammy-chiseled image as a base image for my application which runs in company K8s. When I want to check how I mount volumes to pod container I use kubectl debug but when I run ls /proc/PID/root/ I get access denied because PID runs under user 1654.

Does anybody know how to get to file system of running chiseled container in K8S?

Thank you

**Comment by lbussell** [comment, marked as answer]
[Triage] You will need to access /proc/PID/root while running as a user with the same PID as your Chiseled container. It looks like `kubectl debug` has the option `--as-uid` ([source](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_debug/)). You could try and use that in combination with using one of our runtime-deps images (like `mcr.microsoft.com/dotnet/runtime-deps:8.0-jammy`) as the ephemeral container since it already contains the user `app` with the UID 1654 (you can also access this from outside the image by checking the environment variable `APP_UID`).

**Reply by McMlok** [reply]
Thank you, add user with same UID to busybox image was the magic.
